modules/candidate_parameters/test/TestPlan.md

```diff
@@ -81,3 +81,37 @@
 38. Check that entering a date of death before the date of birth gives a data entry error.
 39. Check that entering a date later than today gives a data entry error.
 40. Check that the date stored matches the _DOD Format_ specified in the configuration module.
+
+## Widgets
+41. Access a candidate in the `candidate_profile` module
+
+### Candidate Info Widget terms
+42. Within the "Candidate Info" card, after the information which
```

Comment:
point 42 : How is the user to test this? 
If it's not testable, it's just pre-amble for what follows, and shouldn't get a number.